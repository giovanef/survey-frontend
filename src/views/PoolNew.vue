<template>
  <div class="pools">
    <pools-form></pools-form>
  </div>
</template>

<script>
import PoolsForm from '@/components/features/PoolsForm.vue';

export default {
  components: {
    PoolsForm,
  },
};
</script>
