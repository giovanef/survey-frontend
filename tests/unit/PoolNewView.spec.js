import { shallowMount } from '@vue/test-utils';

import PoolNew from '@/views/PoolNew.vue';

describe('PoolNewView', () => {
  const build = () => {
    const wrapper = shallowMount(PoolNew);

    return {
      wrapper,
    };
  };

  beforeEach(() => {});

  it('renderiza o componente', () => {
    const { wrapper } = build();

    expect(wrapper.html()).toMatchSnapshot();
  });
});
