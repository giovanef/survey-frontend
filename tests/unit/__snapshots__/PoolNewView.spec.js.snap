// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PoolNewView renderiza o componente 1`] = `
<div>
  <breadcrumbs-stub items="[object Object],[object Object]"></breadcrumbs-stub>
  <div class="pools">
    <div class="title">Adicionar Enquete</div>
    <pools-form-stub></pools-form-stub>
  </div>
</div>
`;
